packages/rc-tooltip/.npmignore

```diff
@@ -0,0 +1,9 @@
+**/*.spec.*
+**/*.story.*
+**/*.tsx
+*.tsx
+readme.tsx
+generated-examples.json
+generated-props.json
```

Comment:
these files are actually not located here anymore, but maybe that's a general cleanup we need to do